services/data-info/src/data_info/routes/avus.clj

```diff
@@ -13,25 +14,30 @@
     (GET* "/avus" [:as {uri :uri}]
       :query [{:keys [user]} StandardUserQueryParams]
       :return AVUGetResult
+      :middlewares [wrap-metadata-base-url]
       :summary "List AVUs (administrative)"
-      :description (str "List iRODS AVUs associated with a data item. Include administrative/system AVUs."
-(get-error-code-block "ERR_NOT_A_USER, ERR_NOT_READABLE"))
+      :description
+          (str "List all AVUs associated with a data item. Include administrative/system iRODS AVUs."
+(get-error-code-block "ERR_NOT_A_USER, ERR_NOT_READABLE")
+(get-endpoint-delegate-block "metadata" "GET /avus/{target-type}/{target-id}"))
       (svc/trap uri meta/admin-metadata-get data-id))
 
-    (POST* "/avus" [:as {uri :uri}]
+    (PUT* "/avus" [:as {uri :uri}]
       :query [{:keys [user]} StandardUserQueryParams]
-      :body [{:keys [irods-avus]} (describe AVUListing "An list of AVUs to add")]
+      :body [body (describe AddMetadataRequest "The iRODS and Metadata AVUs to add")]
       :return AVUChangeResult
+      :middlewares [wrap-metadata-base-url]
       :summary "Add AVUs (administrative)"
-      :description (str "Associate AVUs with a data item. Allow adding any AVU."
-(get-error-code-block "ERR_NOT_A_USER, ERR_NOT_READABLE, ERR_DOES_NOT_EXIST, ERR_NOT_WRITEABLE, ERR_NOT_AUTHORIZED"))
-      (svc/trap uri meta/admin-metadata-add data-id irods-avus))
+      :description (str "Associate iRODS and Metadata AVUs with a data item. Allow adding any AVU."
+(get-error-code-block "ERR_NOT_A_USER, ERR_NOT_READABLE, ERR_DOES_NOT_EXIST, ERR_NOT_WRITEABLE, ERR_NOT_AUTHORIZED")
+(get-endpoint-delegate-block "metadata" "PUT /avus/{target-type}/{target-id}"))
+      (svc/trap uri meta/admin-metadata-add data-id body))
 
     (DELETE* "/avus" [:as {uri :uri}]
       :query [{:keys [user attr value]} AVUDeleteParams]
       :return AVUChangeResult
       :summary "Delete AVU (administrative)"
-      :description (str "Delete a single AVU from a data item. Allows deleting any AVU."
+      :description (str "Delete a single iRODS AVU from a data item. Allows deleting any AVU."
```

Comment:
On second thought, these DELETE endpoints only address one specific AVU by ID at a time (in iRODS we can only address an AVU by attr/value), so I don't think the data-info endpoint should be changed to handle batch deletes, since the `set` endpoint already handles that.

In fact, I don't think these endpoints are in use, so they should probably just be removed for now.